### PR TITLE
Allow simple math input in AmountSpecifyingScreen

### LINKED ISSUE #2728: [feature request] allow basic Math in crafting requests
Feature Request to allow basic math when autocrafting items. Helpfull for bigger projects, i.e. you need 8 upgrades each for 26 pumps.

YPuVXCO

### PR DESCRIPTION
**Pull request by yihualou**
Fixes #2728.

Small additional fix to prevent sending invalid amounts to the server.

### CODE REVIEW

**Comment by mooofins** [comment]
Now has a handy "evaluate" button to immediately see results.

File: src/main/resources/assets/refinedstorage/lang/ru_ru.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
   "misc.refinedstorage.start": "Начать",
   "misc.refinedstorage.clear": "Очистить",
   "misc.refinedstorage.set": "Установить",
+  "misc.refinedstorage.evaluate": "Оценивать",
```
Comment (necauqua) [inline src/main/resources/assets/refinedstorage/lang/ru_ru.json:128]: In russian this word means 'estimate' (like a price of something), the proper word would be 'Вычислить' (which is 'compute', but that's what should be in russian).

source: I'm native speaker

Comment (mooofins) [reply src/main/resources/assets/refinedstorage/lang/ru_ru.json:128]: I knew I was gonna get this wrong dammit. Thanks for catching it!

**Comment by raoulvdberge** [comment]
Hi, thanks for the PR. Can you create a new PR targeting `develop`. Thanks.

**Comment by yihualou** [comment]
Oh woops, done.

**Pull request closed without merging**